src/renderer/routes/test_sequencer_panel/components/modals/CreateSequenceModal.tsx

```diff
@@ -0,0 +1,175 @@
+import { Dialog, DialogContent } from "@/renderer/components/ui/dialog";
+import { Button } from "@/renderer/components/ui/button";
+import { useState } from "react";
+import { Input } from "@/renderer/components/ui/input";
+import { useDisplayedSequenceState } from "@/renderer/hooks/useTestSequencerState";
+import { useCreateSequence } from "@/renderer/hooks/useTestSequencerProject";
+import { InterpreterType } from "@/renderer/types/test-sequencer";
+import { PathInput } from "@/renderer/components/ui/path-input";
+import { useSequencerModalStore } from "@/renderer/stores/modal";
+import { z } from "zod";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { useForm } from "react-hook-form";
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from "@/renderer/components/ui/form";
+
+const formSchema = z.object({
+  name: z
+    .string()
+    .min(1)
+    .max(50)
+    .regex(/\S/, {
+      message: "The sequence name should not contain white spaces",
+    })
+    .transform((val) => val.trim()),
+  description: z.string().max(100),
+  projectPath: z.string().min(1).regex(/\S/, {
+    message: "The project path should not contain white spaces",
+  }),
+});
+
+export const CreateSequenceModal = () => {
+  const { isCreateProjectModalOpen, setIsCreateProjectModalOpen } =
+    useSequencerModalStore();
+  const { elems } = useDisplayedSequenceState();
+  const handleCreate = useCreateSequence();
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [interpreterPath, setInterpreterPath] = useState("");
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [type, setType] = useState<InterpreterType>("flojoy");
```

Comment:
are these just placeholders for later?

Reply:
Yaa, it's to allows seperated env to run the code. It's half done at the moment. 
- It's not new, they look new because I renamed the file.
- I could removed them tho if needed